Vagrantfile

```diff
@@ -5,43 +5,38 @@ require 'yaml'
 require "#{File.dirname(__FILE__)}/lib/forklift"
 
 VAGRANTFILE_API_VERSION = '2'
-SUPPORT_SSH_INSERT_KEY = Gem.loaded_specs['vagrant'].version >= Gem::Version.create('1.7')
-SUPPORT_NAMED_PROVISIONERS = Gem.loaded_specs['vagrant'].version >= Gem::Version.create('1.7')
-SUPPORT_BOX_CHECK_UPDATE = Gem.loaded_specs['vagrant'].version >= Gem::Version.create('1.5')
```

Comment:
I recall that Vagrant 1.8 was needed for Ansible variables. Since the pipelines now use Ansible variables as well, I reasoned that it was a global hard require.